src/content/docs/webdev-hosting/Netlify/netlify-case-study.md

```diff
@@ -0,0 +1,107 @@
+
+- Continuous Deployment: Netlify has a feature that supports continuous deployment, by
+  connecting your repository, Netlify will automatically build, deploy and update the site whenever changes are made.
+
+- Analytics and Monitoring: This feature is really awesome it lets you track your monitor traffic, site performance and gives you good insight into user behavior. Its essetiantly a tool that´ll help developers get better performance, optimazation and user experience.
+
+## Strengths
+
+- Ease of Use: As said ealier, Netlify is already known for its user-friendly interface and workflows. With just a few clicks developers can easily deploy their projects with Git intergration.
+
+- Serverless Functions: With Serverless Functions it gives developers the ability to execute server-side logic without managing infrastructure.
+
+- Continuous Deployment: Netlify supports continuous deployment, by
+  connecting your repository, Netlify will automatically build, deploy and update the site whenever changes are made.
+
+- Developer-Friendly: Netlify is very user-friendly and really easy to dive in too with all the documentation, tutorials and resources that are available for developers. And by doing this it lets developers get started quickly and makes troubleshooting a lot easier aswell.
+
+## Weaknesses
+
+- Cost: Even though Netlify is very known for being really good and generous with its free basic features, with that being said the costs can get really high for organzations that may have a lot of traffic and so because of this additional costs for premium features may occur.
+
+- Complexity for Large-Scale Applications: Since Netlify is more suited for small applications, you may find it lack more advanced features and options for large scale
+  applications. Beacuse of this organzations may choose other options for hosting instead.
+
+- Limited Backend: As stated earlier Netlify relies on serverless functions, because of this it may limit amount of backend customization and control available to developers.
+
+- Third-Party Services: Third-Party Services is a big part of netlify, it relies on services like AWS and and CDN services. The big risk with this is that this may introduce dependencies which then makes uptime and reliability affected.
+
+## Market Comparison
+
+If compared to a more flexible and advanced hosting website like Heroku, I would say that Netlify excels and shines the most when it comes to hosting static websites and fast deployment while offering great features such as Git Intergrations, Forms Handling and serverless functions.
+
+While on the other hand we have heroku, which is pretty much a lot like Netlify besides the fact that its more advanced and more flexible. Heroku provides good support for applications that takes use of full-stack web applications with server-side logic and databases.
```

Comment:
Same thing here. Fix the tone and grammar. I recommend you use tools like quillbot.com